assets/src/app.js

```diff
@@ -76,6 +76,40 @@ function App() {
     }
   }, []);
 
+  // Loop allresources and set filter options
+  useEffect(() => {
+    if (resourcesOptions.length === allResources.length) {
+      return;
+    }
+    const locations = [];
+
+    Object.values(allResources).forEach((value) => {
```

Comment:
```suggestion
const locations = [...new Set(allResources.map(item => item.location))];
```

Reply:
Forslaget giver ['LOCATION1', 'LOCATION2', undefined, 'NEW LOCATION']
Har ændret det til følgende, som stadig benytter Set, hvilket er favre nye verden for mig. Sejt!

const locations = [...new Set(
      allResources
      .filter(resource => resource.location !== "")
      .map(resource => resource.location)
      )]